CHANGELOG.md

```diff
@@ -32,6 +32,7 @@
 - [Redo stack is no longer lost when interacting with text literals][11908].
 - Symetric, transitive and reflexive [equality for intersection types][11897]
 - [IR definitions are generated by an annotation processor][11770]
+- [Introduce ENSO_LAUNCHER env var to control build of native image][12035]
```

Comment:
I don't think this deserves a CHANGELOG note. It is not change visible by end users.

Reply:
CHANGELOG reverted in c3eace876c11eb00da00dc17850a0900a7f1b418